src/components/RefineSearch/RefineSearch.tsx

```diff
@@ -10,6 +10,8 @@ import { useState, useCallback } from "react"
 import { useRouter } from "next/router"
 
 import styles from "../../../styles/components/Search.module.scss"
+import type { DateFormName } from "../SearchFilters/FieldsetDate"
```

Comment:
Small nit: you can combine these into one import statement:

import FieldsetDate, { type DateFormName } from "../SearchFilters/FieldsetDate"